web-admin/src/routes/[organization]/[project]/-/deployment/+page.svelte

```diff
@@ -1,26 +1,23 @@
 <script lang="ts">
   import { page } from "$app/stores";
-  import { createAdminServiceGetProject } from "../../../../../client";
-  import Logs from "../../../../../components/deployments/Logs.svelte";
-  import Status from "../../../../../components/deployments/Status.svelte";
-
-  const proj = createAdminServiceGetProject(
-    $page.params.organization,
-    $page.params.project
-  );
+  import ProjectDeploymentLogs from "../../../../../components/projects/ProjectDeploymentLogs.svelte";
```

Comment:
do we not have any aliases for these? If not, it might be advantageous to set up a few in this project.